docker/manifests/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Lets remove this from here and keep the Kubernetes manifests in a separate repo.